src/main/webapp/i18n/de/metis.json

```
@@ -104,6 +104,11 @@
             "answerPost": {
                 "created": "Antwort erfolgreich erstellt",
                 "deleted": "Antwort erfolgreich gelöscht"
+            },
+            "userAuthorityTooltips": {
+                "instructor": "Lehrkraft: Verantwortlich für die Kursdurchführung.",
```

```suggestion
                "instructor": "Lehrender: Verantwortlich für die Kursdurchführung.",
```
Or you could also think about `Dozent`

---

Lehrkraft is the explicit example in the docs for gender neutral language: https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/#german-language

Lehrender does not fulfil that requirement.